### PR TITLE
Upgraded Pass-Retrieval-Tool

### PR DESCRIPTION
**Pull request by Warafux**
-POO (and refactoring of methods / variables)
-MultiLingual (Works on english & spanish cmd)
-Json output

### CODE REVIEW

**Comment by Warafux** [comment]
I've noticed a bug, FIXING IT
This is only working on python 2.7.x, is that okay?

**Comment by JeeveshN** [comment]
Yeah that's fine.
Fix the bug and send the PR adain.

File: README.md [hunk 1]
```diff
@@ -12,7 +29,22 @@ git clone https://github.com/JeeveshN/Retrieve-Wifi-Passwords.git or download th
...
+{
+  "Example-Wifi-SSID" : "WifiPassword1234",
+  "WifiSSID2" : "p4$$w0rD"
+}
+
+```
+
+
+### Retrieve Based on Wifi Name [REMOVED] 
+Removed because it is not useful at all.
```
Comment (JeeveshN) [inline README.md:47]: I would like you to keep this functionality.

**Comment by Warafux** [comment]
Alright, sending pull asap.

File: README.md (hunk 1 above)
Comment (Warafux) [reply README.md:47]: Okay

**Comment by Warafux** [comment]
New Pull Request sent.

**Pull request closed without merging**